traveler-swift-core/TravelerKit/Models/Details/Historic/PurchasedParkingItemDetails.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  PurchasedParkingItemDetails.swift
+//  TravelerKit
+//
+//  Created by Rakin Hoque on 2019-12-03.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+// TODO: Make decodable once backend changes to support single model aggregation
+/// The historic information of a `ParkingItem` combined with `PurchasedParkingProduct`
+public struct PurchasedParkingItemDetails: CatalogItemDetails, PurchasedProduct {
```

Comment:
Same comment as PurchasedBookingItemDetails. The current use for our details models is for display purposes only. I also believe that our PurchaseDetails models (i.e PurchasedBookingItemDetails, PurchasedParkingItemDetails, PurchasedPartnerOffering...) don't need to conform to any protocol and should be stand alone entities by themselves. They're for sure not CatalogItemDetails because they don't appear in the catalog.

Reply:
Added TODO to remove the PurchasedProduct conformance